### ISSUE TITLE
Does multiple data parallel alignment share hbndb?

### ISSUE THREAD

**Issue by chenzemin** [chenying2016/queries#28]
I used snakemake pipeline to analyze my data.
Multiple data sets were analyzed in parallel, but I did not use the "-db_dir" argument.

It appears that each data needs to be split  the database and written into the same default hbndb. Does this have any impact? Because I am using the nt database, this process is expected to take a significantly long time.

**Comment by chenying2016** [comment]
Yes, you should place different parts of nt into different directories to avoid this problem.

**Comment by chenzemin** [comment]
Are there any other options available? It seems that the v2.0.0 version doesn't support pre-building the index. I have a large number of samples, and if I have to re-read the nt database for each sample and store it on the disk, it would be a significant challenge in terms of time and storage space.

**Comment by chenying2016** [comment]
The index is no longer used in v2. v2 no longer takes time on building the index.  db_dir just stores the compressed sequences.
